installer.sh

```diff
@@ -400,11 +401,109 @@ case ${ANSWER:0:1} in
     else
       git clone https://github.com/cve-search/cve-search.git external/cve-search
     fi
+    cd ./external/cve-search/ || exit 1
+    pip3 install -r requirements.txt
+    xargs sudo apt-get install -y < requirements.system
     echo -e "\\n""$MAGENTA""$BOLD""For using CVE-search you have to install all the requirements and the needed database.""$NC"
     echo -e "$MAGENTA""$BOLD""Installation instructions can be found on github.io: https://cve-search.github.io/cve-search/getting_started/installation.html#installation""$NC"
   ;;
 esac
 
+# binwalk
+
+INSTALL_APP_LIST=()
+print_tool_info "python3-pip" 1
+print_tool_info "python3-crypto" 1
+print_tool_info "python3-opengl" 1
+print_tool_info "python3-pyqt5" 1
+print_tool_info "python3-pyqt5.qtopengl" 1
+print_tool_info "python3-numpy" 1
+print_tool_info "python3-scipy" 1
+print_tool_info "mtd-utils" 1
+print_tool_info "gzip" 1
+print_tool_info "git" 1
+print_tool_info "bzip2" 1
+print_tool_info "tar" 1
+print_tool_info "arj" 1
+print_tool_info "lhasa" 1
+print_tool_info "p7zip" 1
+print_tool_info "p7zip-full" 1
+print_tool_info "cabextract" 1
+print_tool_info "cramfsswap" 1
+print_tool_info "squashfs-tools" 1
+print_tool_info "sleuthkit" 1
+print_tool_info "default-jdk" 1
+print_tool_info "lzop" 1
+print_tool_info "srecord" 1
+print_tool_info "build-essential" 1
+print_tool_info "zlib1g-dev" 1
+print_tool_info "liblzma-dev" 1
+print_tool_info "liblzo2-dev" 1
+print_tool_info "firmware-mod-kit" 1
+
+echo -e "\\n""$MAGENTA""$BOLD""Do you want to download and install binwalk, yaffshiv, sasquatch, jefferson, unstuff, cramfs-tools and ubi_reader (if not already on the system)?""$NC"
+read -p "(y/N)" -r ANSWER
+case ${ANSWER:0:1} in
```

Comment:
Please add here the option for the forced installation, like the one from above:
```
if [[ "$FORCE" -eq 0 ]] ; then
  echo -e "\\n""$MAGENTA""$BOLD""Do you want to download and install the net-tools, pip3, cve-search and cve_searchsploit (if not already on the system)?""$NC"
  read -p "(y/N)" -r ANSWER
else
  echo -e "\\n""$MAGENTA""$BOLD""net-tools, pip3, cve-search and cve_searchsploit (if not already on the system) will be downloaded and be installed!""$NC"
  ANSWER=("y")
fi
```